mypy/build.py

```diff
@@ -569,6 +571,7 @@ class BuildManager:
       flush_errors:    A function for processing errors after each SCC
       saved_cache:     Dict with saved cache state for coarse-grained dmypy
                        (read-write!)
+      cache_enabled:   Whether cache usage is enabled
```

Comment:
Could be worth mentioning how this relates to `use_fine_grained_cache` -- the latter means that we attempt to use the cache if it exists, and this indicates whether we actually used it or something.